test/CMakeLists.txt

```diff
@@ -200,6 +200,7 @@ set_source_files_properties(includes.cpp PROPERTIES COMPILE_DEFINITIONS
 if (NGRAPH_UNIT_TEST_OPENVINO_ENABLE)
     message(STATUS "NGRAPH_UNIT_TEST_OPENVINO_ENABLE = TRUE")
     add_definitions(-DNGRAPH_UNIT_TEST_OPENVINO_ENABLE)
+    set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -fPIC")
```

Comment:
suggest to use `CMAKE_POSITION_INDEPENDENT_CODE` which is cross platform.

Reply:
Removed this line.